### PR TITLE
Rename "presenting browsing context" to "receiving browsing context".

### PR DESCRIPTION
**Pull request by mounirlamouri**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -450,7 +450,7 @@
       </section>
       <section>
         <h3>
-          Presenting context: monitor available session(s) and say hello.
+          Receiving context: monitor available session(s) and say hello.
```
Comment (mfoltzgoogle) [inline index.html]: At this point I don't believe "Receiving context" has been defined yet in the prose?

File: index.html [hunk 2]
```diff
@@ -514,10 +514,10 @@
           <a>presentation session</a> via a <code>sessionconnect</code> event.
         </p>
         <p>
-          The <dfn>presenting browsing context</dfn> (or <dfn data-lt=
+          The <dfn>receiving browsing context</dfn> (or <dfn data-lt=
           "presentation context">presentation</dfn> for short) is the browsing
```
Comment (mfoltzgoogle) [inline index.html]: Do we need a definition of "presentation context" any more?

File: index.html [hunk 3]
```diff
@@ -1479,9 +1478,9 @@
                 <li>Set the <a>presentation session identifier</a> of
                 <var>S</var> to <var>I</var>.
                 </li>
-                <li>Establish the connection between the controlling and
-                presenting browsing contexts using an implementation specific
-                mechanism.
+                <li>Establish the connection between the controlling and <a data-lt=
```
Comment (mfoltzgoogle) [inline index.html]: Maybe define this term above as "receiving browsing context|receiving browsing contexts", similar to the controlling case.

**Comment by mfoltzgoogle** [comment]
LGTM with comments applied.  Thanks for doing this @mounirlamouri!

File: index.html (hunk 1 above)
Comment (mounirlamouri) [inline index.html]: Done.

File: index.html (hunk 2 above)
Comment (mounirlamouri) [inline index.html]: Doesn't seem like it. Removed.

File: index.html (hunk 3 above)
Comment (mounirlamouri) [inline index.html]: Done.

**Comment by mounirlamouri** [comment]
Review comments applied. Merging.

**Pull request merged**